chart/config/proxy/lib.locations.conf

```diff
@@ -43,8 +43,19 @@ location / {
     add_header "Last-Modified" "";
 
     proxy_pass http://dashboard$request_uri;
+
+    # # Page Not Found
+    # Let's redirect the browser agent to the web app, if no resources are found in the locations specified here.
+    # 
+    # This is derived from the issue https://github.com/gitpod-io/gitpod/issues/2659, which
+    # was appearently caused by Twitter's link shortening. So, if the leading www is missing
+    # in the URL by accident, we will redirect to the web on 404 – Not Found.
+    error_page 404 @error_404_dashboard;
 }
 
+location @error_404_dashboard {
+    return 302 https://www.${PROXY_DOMAIN}$request_uri;
```

Comment:
I wonder if the assumption that there is a `www.` subdomain is new - I bet not. Also, would be nice if this is configurable, as "landing page", e.g.
But I think this is fine for now, especially as we want to test if this breaks sth on staging.